.github/merge_request_template/Default.md

```diff
@@ -0,0 +1,26 @@
+## Describe your changes
+
+### JIRA link
+<!--- Please include a link to the JIRA ticket if there is one associated with -->
+
+### Design document
+<!--- Please include a link to the design document if there is one 
+associated with these changes-->
+
+### What's new?
+<!--- Please describe the changes in this PR -->
+
+### How to test?
+<!--- Please describe how to test these changes -->
+
+## PR Author Check List
+
+* [ ]  All commit messages adhere to the project [standard](https://github.com/openshift/rosa/blob/master/CONTRIBUTING.md) 
+
+* [ ]  All commits are squashed into a single commit `git rebase -i HEAD~<number of commits>`
+
+* [ ]  All code changes have unit test coverage
+
+* [ ]  All changes have been tested locally and the steps documented in the `How to Test?` section
+
+* [ ]  All required API changes are deployed to production
```

Comment:
We do not require this, this is a requirement of the Release, not merging to master